src/components/criminals.tsx

```diff
@@ -12,7 +14,10 @@ const criminalsTypes = {
 
 export const Criminals = () => {
   const [open, setOpen] = useState(false);
-  const { data } = api.criminal.getAll.useQuery(undefined, { enabled: open });
+  const { data, isLoading } = api.criminal.getAll.useQuery(undefined, {
```

Comment:
why are you sending undefined?

Reply:
if I wont do it I cannot access to the options